scripts/wallet.js

```diff
@@ -794,10 +792,11 @@ export class Wallet {
      * out of the vin. This will hapèen after the mempool refactor,
      * But for now we can just recalculate the UTXOs
      */
```

Comment:
```suggestion
     * out of the vin. This will happen after the mempool refactor,
     * But for now we can just recalculate the UTXOs
     */
```

Add `target` param to the JSDoc of `getUTXOsForShield`, also spotted a small typo spotted in the existing JSDoc.